roles/ovirt-cluster-upgrade/README.md

```diff
@@ -0,0 +1,60 @@
+oVirt cluster upgrade
+=========
+
+This role will iterate through all hosts in cluster and upgrades them.
+
+Requirements
+------------
+
+ * Ansible version 2.3 or higher
+ * Python SDK version 4 or higher
+
+Role Variables
+--------------
+
+| Name               | Default value         |                                                     |
+|--------------------|-----------------------|-----------------------------------------------------|
+| cluster_name       | Default               | Name of the cluster to be upgraded.                 |
+| stopped_vms        | UNDEF                 | List of VM names to be stopped before upgrade.      |
+| force_pinned       | false                 | If true the pinned to host VMs will be stopped.     |
```

Comment:
It's no obvious from my side what this parameter does from the 1st look, I'd prefer 'stop_pinned_to_host_vms'. I know it's long :-) but it's not an issue from yml file and everybody know what this parameter does from its name.